### ISSUE TITLE
Entries marked with "Unknown Editor" and wrong OS on WakaTime

### ISSUE THREAD

**Issue by mai-soup** [vanBassum/Wakatime.Unity#4]
![image](https://user-images.githubusercontent.com/97809168/191785816-2d6d6259-7b74-4340-a9e0-eb125396eaba.png)

See screenshot. At first I chalked up 1m in Linux to be something I must've opened on accident earlier there, but it is increasing along with the "Unknown Editor" entry and Unity language entry, and I'm only running Unity on Windows, so it should be tagged as the OS I'm using.

**Comment by vanBassum** [comment]
Not sure if the useragent is what its supposed to be, although is seems to be fixing this problem

**Comment by Rabadash8820** [comment]
@vanBassum I don't believe this Issue was actually resolved. As of the current commit, the WakaTime dashboard is still showing "Unknown Editor". I suspect that it's because the code that sets the `Plugin` CLI parameter has been [commented out](https://github.com/vanBassum/Wakatime.Unity/blob/main/Editor/Handlers/WakatimeCliHandler.cs#L32).

**Comment by vanBassum** [comment]
Yes, I think you are right, I'll see what happens when I uncomment that line.
